app/main.py

```diff
@@ -23,6 +24,26 @@ class StatusBody(BaseModel):
     processing_status: str | None = None
     message: str | None = None
 
+class NeoFourJ:
+
+    def __init__(self, uri, user, password):
+        self.driver = GraphDatabase.driver(uri, auth=(user, password))
+
+    def close(self):
+        self.driver.close()
+
+    def get_rank(self, param):
+        with self.driver.session() as session:
+            rank = session.write_transaction(self._get_rank, param)
+            return rank
+
+    @staticmethod
+    def _get_rank(tx, param):
+        result = tx.run(
+            'MATCH (parent:Taxonomies)-[:CHILD]->(child:Taxonomies) where parent.name=~' '"' '.*' + param + '.*' '"'
+                                                                                                            'RETURN parent')
+        return result.single()[0]
+
```

Comment:
can you move it to the separate file please

Reply:
Resolved